### ISSUE TITLE
Inconsistently failing tests

### ISSUE THREAD

**Issue by jwillenbring** [trilinos/Trilinos#7378]
## Bug Report
@trilinos/framework 

### Description

A number of tests fail occasionally for PR and other testing. Sometimes things just go wrong with a test run (network or machine issue, etc) and something fails, but there are some tests now, and some show up from time to time, that do consistently fail some percentage of the time less than 100 and more than 0. These are hard to track down. When the PR testing runs a second time after a failure, for example, a person might naturally assume without looking closely that the cause of the previous failure was a system or network issue. When a test fails only once, and the failure cannot reliably be reproduced, it is hard for a person to know if they should file a bug, and for the recipient of the bug to know how to proceed.

The purpose of this issue is to serve as a central place for people to log failing tests. This will make the names of failing tests more easily searchable, and may make it more clear under what conditions tests are more likely to fail and with what frequency failures are occurring. If a test fails a few times, we can extract whatever information we can, including links to the failure occurrences, and file a separate ticket. 

This is meant to be a running issue to track existing and new failures, not an issue that can be fully "resolved". 

To help people notice occasional failing tests more easily, I am going to revisit the number of times a retry occurs automatically for PR testing. I will also send out a note to the developer list to announce this strategy of trying to clean up occasionally failing tests.

**Comment by jwillenbring** [comment]
I observed that this test failed, then passed in the next run for my PR:

[MueLu_UnitTestsTpetra_MPI_4](https://testing.sandia.gov/cdash/testDetails.php?test=112405591&build=7032651)

Here is the [associated build](https://testing.sandia.gov/cdash/viewTest.php?buildid=7032651)

Here is a [summary of the test runs for that day](https://testing.sandia.gov/cdash/testSummary.php?project=1&name=MueLu_UnitTestsTpetra_MPI_4&date=2020-05-15) for that test. There were some other failures, but none of the rest were time-outs.

**Comment by wadeburgess** [comment]
As per Jim W, closing this issue.
